.golangci.yml

```diff
@@ -0,0 +1,307 @@
+  govet:
+    # report about shadowed variables
+    check-shadowing: true
+    # settings per analyzer
+    settings:
+      printf: # analyzer name, run `go tool vet help` to see all analyzers
+        funcs: # run `go tool vet help printf` to see available settings for `printf` analyzer
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Infof
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
+    # enable all analyzers
+    enable-all: true
+  depguard:
+    list-type: blacklist
+    include-go-root: false
+    packages:
+      - github.com/sirupsen/logrus
+    packages-with-error-message:
+      # specify an error message to output when a blacklisted package is used
+      - github.com/sirupsen/logrus: "logging is allowed only by logutils.Log"
+  lll:
+    # max line length, lines longer will be reported. Default is 120.
+    # '\t' is counted as 1 character by default, and can be changed with the tab-width option
+    line-length: 120
+    # tab width in spaces. Default to 1.
+    tab-width: 1
+  maligned:
+    # print struct with more effective memory layout or not, false by default
+    suggest-new: true
+  nakedret:
+    # make an issue if func has more lines of code than this setting and it has naked returns; default is 30
+    max-func-lines: 30
+  testpackage:
+    # regexp pattern to skip files
+    skip-regexp: (export|internal)_test\.go
+  unused:
+    # treat code as a program (not a library) and report unused exported identifiers; default is false.
+    # XXX: if you enable this setting, unused will report a lot of false-positives in text editors:
+    # if it's called for subdir of a project it can't find funcs usages. All text editor integrations
+    # with golangci-lint call it on a directory with the changed file.
+    check-exported: false
+  whitespace:
+    multi-if: false   # Enforces newlines (or comments) after every multi-line if statement
+    multi-func: false # Enforces newlines (or comments) after every multi-line function signature
+  gci:
+    local-prefixes: "bitbucket.org"
+  misspell:
+    #locale: US
+    ignore-words:
+  custom:
+  # Each custom linter should have a unique name.
+
+linters:
+  enable:
+    - gofmt
+    - govet
+    # - errcheck
+    # - staticcheck
+    # - unused
+    # - gosimple
+    - ineffassign
+    - typecheck
+    # Additional
+    # - lll
+    - godox
+    #- gomnd
+    #- goconst
+    # - gocognit
+    # - maligned
+    # - nestif
+    # - gomodguard
+    - nakedret
+    # - gci
+    # - misspell
+    # - gofumpt
+    - whitespace
+    # - unconvert
+    - predeclared
+    # - noctx
+    - dogsled
+    # - bodyclose
+    - asciicheck
+      #- stylecheck
+      # - unparam
+    #- wsl
```

Comment:
Are you planning to open other PRs to enable additional linters?

Reply:
Yes, we need to activate disabled linters one by one.

Reply:
Ok, thanks!